conf/test_doubleprimers.config

```diff
@@ -1,19 +1,23 @@
 /*
- * -------------------------------------------------
- *  Nextflow config file for running tests
- * -------------------------------------------------
- * Defines bundled input files and everything required
- * to run a fast and simple test. Use as follows:
- *   nextflow run nf-core/ampliseq -profile test_doubleprimers,<docker/singularity>
- */
+========================================================================================
+    Nextflow config file for running minimal tests
+========================================================================================
+    Defines input files and everything required to run a fast and simple pipeline test.
```

Comment:
I suppose we could say more here as it's not just the standard minimal test, but not sure it's so important.

Reply:
True, that could be adjusted.

Reply:
Hm, actually, the header is distinguishing between "minimal" and "full" test dataset, so that might be ok. The important lines are in my opinion
```
    config_profile_name = 'Test doubleprimers profile'
    config_profile_description = 'Minimal test dataset to check pipeline function when removing double primers'
```
see [here](https://github.com/d4straub/ampliseq/blob/224fd3cbba3d5edab44beb2e172cfe5205d356e2/conf/test_doubleprimers.config#L14-L15). Those lines are also displayed in the console when running the test profile.